src/org/parosproxy/paros/view/FindDialog.java

```diff
@@ -65,10 +73,14 @@ public void setLastInvoker(JTextComponent lastInvoker) {
 
     /**
      * @throws HeadlessException
+     * @deprecated TODO add version
+     * @see {@link #FindDialog(Frame, boolean)}

```

Comment:
Seems to be a wrong auto-complete suggestion then, I get warnings with JavaDoc and Eclipse:
```
FindDialog.java:77: error: unexpected content
     * @see {@link #FindDialog(Frame, boolean)}
       ^
FindDialog.java:78: error: unexpected content
     * @see {@link #FindDialog(Window, boolean)}
       ^
FindDialog.java:81: warning - Tag @see:illegal character: "123" in "{@link #FindDialog(Frame, boolean)}"
FindDialog.java:81: warning - Tag @see:illegal character: "64" in "{@link #FindDialog(Frame, boolean)}"
FindDialog.java:81: warning - Tag @see:illegal character: "123" in "{@link #FindDialog(Window, boolean)}"
FindDialog.java:81: warning - Tag @see:illegal character: "64" in "{@link #FindDialog(Window, boolean)}"

```